src/cdk/tree/tree.ts

```diff
@@ -376,12 +375,12 @@ export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
         this._setRoleFromChildren(childrenNodes as T[]);
       } else if (isObservable(childrenNodes)) {
         childrenNodes.pipe(takeUntil(this._destroyed))
-            .subscribe(children => this._setRoleFromChildren(children));
+          .subscribe(children => this._setRoleFromChildren(children));
       }
     }
   }
 
   protected _setRoleFromChildren(children: T[]) {
```

Comment:
@annieyw This function doesn't seem necessary now that it's just a simple one-liner to set the role, and it doesn't use the function parameters.